config/routes.rb

```diff
@@ -12,7 +13,7 @@
       registrations: 'users/registrations',
       omniauth_callbacks: 'users/omniauth_callbacks'
     }
-  resources :users, only: [:show]
```

Comment:
Is this just a left-over you removed or removed by accident?

Reply:
This is a leftover.
There used to be a users controller only for the show action. It has become obsolete since #55.